README.md

```diff
@@ -64,21 +78,17 @@ CSS
 
 IGNORE INLINE STYLE
 .color-picker
-
 ```
 - `INVERT` rule inverts specified elements.
-For **Dynamic mode** use `INVERT` only for dark images, that are invisible on dark backgrounds (icons, diagrams, charts, `<img>` and `<svg>` elements).
+For **Dynamic mode** use `INVERT` only for dark images that are invisible on dark backgrounds (icons, diagrams, charts, `<img>` and `<svg>` elements).
 - `CSS` rule adds custom CSS to a web page.
 `!important` keyword should be specified for each CSS property to prevent overrides by other stylesheets.
 **Dynamic mode** supports `${COLOR}` template, where `COLOR` is a color value before the inversion (`white` will become `black` in dark mode).
-- `IGNORE INLINE STYLE` rule will not analyze the inline style of matched selector's. (e.g. --darkreader-inline-bgcolor or <p style='color: red'>This will not be analyzed by dark reader if it has an matched selector</p>)
-- **Special notice for Filter and Filter+ config**.
```

Comment:
Why removing this section?

Reply:
It's still there.  It's just moved to a more appropriate location.  Also,  the `<p>` caused rendering problems in the GitHub web-app, so I fixed that by using proper escaping.

Reply:
Didnt' saw that.

Reply:
:)